js/src/launchAsync.ts

```diff
@@ -203,7 +205,7 @@ export function launchAsync(token: string, subdomain: string, content: Content,
                         container: iframeContainer,
                         sessionId: response.sessionId,
                         charactersProcessed: response.meteredContentSize,
-                        postLaunchOperations: {
+                        postLaunchOperations: options.providePlayPauseControl === true && {
```

Comment:
Still need to return null if false. Or does this work if not true?

Reply:
it works but getting false instead of null
![image](https://user-images.githubusercontent.com/87383938/144328440-bb5f3415-3759-48d6-b163-6756df104bd8.png)

Reply:
done